mod/wikirate_source/set/type_plus_right/source/file/file.rb

```diff
@@ -1,3 +1,6 @@
+card_accessor :metric, :type=>:pointer
+card_accessor :year, :type=>:pointer
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator `=>`.